source/Mission.cpp

```diff
@@ -226,6 +226,8 @@ void Mission::Load(const DataNode &node)
 			clearance = (child.Size() == 1 ? "auto" : child.Token(1));
 			clearanceFilter.Load(child);
 		}
+		else if(child.Size() == 2 && child.Token(0) == "ignore" && child.Token(1) == "clearance")
```

Comment:
```suggestion
		else if(child.Size() == 2 && child.Token(0) == "ignore" && child.Token(1) == "clearance")
```
unless we want to expand on this with other ignores, this should be one token

Reply:
Yes, we will expand with more ignores: keystone, gaslining, etc. Those are harder to implement, while "clearance" is trivial.

Reply:
well in that case, I am fine.

Although... how about allowing it to be a child node? On the other hand that makes not a lot of sense if we have only one, probably something to be implemented later.

Reply:
Yes, I expect people will want all sorts of additions one day. For now, I want to limit it to the minimum necessary for the feature, and avoid scope creep.

Reply:
perfectly fine then.